release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -28,13 +28,375 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
+    [int, boolean, anydata...] v2 = [ ... t2, "x", "y"]; // works as all fixed tuple members are guaranteed to have been provided values
+}
+```
+
+##### Allow `int*float`, `float*int`,` int* decimal`, `decimal*int`, `float/int`, `decimal/int`, `float%int`, and `decimal%int`.
+
+Multiplicative expression is now allowed with `int`, `float` and `int`, and `decimal` operands.  The resulting type of the expression will be the fractional type.
+
+This allows the below.
+- Multiplication supports both `int*float` and `float*int` similar to `decimal*float` and `decimal*int`
+- For division and modulo, only the `floating-point` operand is supported as the dividend (i.e., `float/int`, `decimal/int`, `float%int`, and `decimal%int` are supported
+
+```ballerina
+import ballerina/io;
+
+public function main() {
+    int quantity = 5;
+    float weight = 2.5;
+    decimal unitPrice = 10.55;
+
+    float totalWeight = weight * quantity;
+    io:println(totalWeight); // 12.5
+    decimal totalPrice = unitPrice * quantity;
+    io:println(totalPrice); // 52.750
+
+    float weightInGrams = 2456;
+    int gramsPerKG = 1000;
+    float weightInKG = weightInGrams / gramsPerKG;
+    io:println(weightInKG); // 2.456
+
+    decimal totalAmount = 1000.5;
+    int numberOfPersons = 3;
+    decimal remainingAmount = totalAmount % numberOfPersons;
+    io:println(remainingAmount); // 1.5
+}
+```
+
+##### New lang library functions
+
+###### New `lang.array:some()` function
+
+The `lang.array:some()` function tests whether a function returns `true` for some member of an array. 
+
+```ballerina
+import ballerina/io;
+
+function func1(int i) returns boolean {
+    return i > 2;
+}
+
+public function main() {
+    io:println([1, 3].some(func1)); // true
+    io:println([1, -3].some(func1)); // false
+}
+```
+###### New `lang.array:every()` function
+
+The `lang.array:every` function tests whether a function returns `true` for every member of an array.
+
+```ballerina
+import ballerina/io;
+
+function func1(int i) returns boolean {
+    return i > 2;
+}
+
+public function main() {
+    io:println([5, 3].every(func1)); // true
+    io:println([1, 3].every(func1)); // false
+}
+```
+###### New `lang.decimal:quantize()` function
+
+The `lang.decimal:quantize()` function has been introduced to control the precision of decimal values, which return a value equal to the first operand after rounding and having the exponent of the second operand.
+
+```ballerina
+import ballerina/io;
+
+public function main() {
+    io:println(decimal:quantize(123.123, 1.0)); // 123.1
+    io:println(decimal:quantize(123.123, 1.00)); // 123.12
+    io:println(decimal:quantize(123.123, 1.000)); // 123.123
+}
+```
+If the length of the coefficient after the quantize operation is greater than the precision, the function results in a panic.
+
+```ballerina
+public function main() {
+    decimal _ = decimal:quantize(123.1233, 1E-36); // results in a panic
+}
+```
```

Comment:
This doesn't come from the spec, right? Do we need to specifically mention this?

Reply:
@chiranSachintha

Reply:
@MaryamZi It is not specifically mentioned in Ballerina spec. But its mentioned in the IEEE spec.

Reply:
This is in the IEEE spec(for `quantize` function we use spec related to that).